ssl/statem/statem_lib.c

```diff
@@ -169,9 +169,13 @@ int tls_setup_handshake(SSL *s)
 static int get_cert_verify_tbs_data(SSL *s, unsigned char *tls13tbs,
                                     void **hdata, size_t *hdatalen)
 {
+#ifdef CHARSET_EBCDIC
+    static const char *servercontext = { 0x54 ,0x4c ,0x53 ,0x20 ,0x31 ,0x2e ,0x33 ,0x2c ,0x20 ,0x73 ,0x65 ,0x72 ,0x76 ,0x65 ,0x72 ,0x20 ,0x43 ,0x65 ,0x72 ,0x74 ,0x69 ,0x66 ,0x69 ,0x63 ,0x61 ,0x74 ,0x65 ,0x56 ,0x65 ,0x72 ,0x69 ,0x66 ,0x79, 0x00 };
```

Comment:
Same issue with spaces and commas here too.

The space/comma thing isn't consistent.

Reply:
Fixing